libmscore/cmd.cpp

```diff
@@ -2745,6 +2745,8 @@ void Score::cmdSlashFill()
             for (Segment* s = startSegment; s && s->tick() < endTick; s = s->next1()) {
                   if (s->segmentType() != SegmentType::ChordRest)
                         continue;
+                  if (s->measure()->isMMRest())
+                        continue;
```

Comment:
1. Why two if statements. Why not this?

`if (s->segmentType() =! SegmentType::ChordRest || s->measure()->isMMRest())
      continue;`

2. Maybe it will be useful to create isChordRestSegment() and similar functions for SegmentType (found [here](https://github.com/musescore/MuseScore/blob/master/libmscore/types.h#L287) similar to those created for ElementType [here](https://github.com/musescore/MuseScore/blob/master/libmscore/scoreElement.h#L248) in segment.h